### DISCUSSION TITLE
Common abbreviation dictionary

### DISCUSSION THREAD

**Discussion by ghost** [AnySoftKeyboard/AnySoftKeyboard#2647] [Ideas]
Hi!
Thanks for developing AnySoftKeyboard. I really like it.
I have one feature request:
* To be able to have a common abbreviation dictionary for all languages. It is a big task to have to type everything again for every keyboard layout.

**Comment by menny** [comment]
This is a good idea. And I think that the code supports that, but the UI doesn't..
I suggest you create a ticket for this and we'll work on it for r5
